overlord/hookstate/ctlcmd/model_test.go

```diff
@@ -0,0 +1,421 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
```

Comment:
Nice tests! Could you also add test to cover running `snapctl model` command from the snap (as opposed to running from a hook)? You can look at some of the existing tests e.g. `TestRefreshProceedFromSnap`, it's mostly about not having a hook task when creating `HookContext` and not having hook handler.

Reply:
Really great suggestion, I've added this!